main.py

```diff
@@ -2,7 +2,8 @@
 
 from shapeAnalysisProject.input.x_ray import read_image
 from shapeAnalysisProject.image_filtering.x_ray.Sobel import filter_image
-from shapeAnalysisProject.shape_matching.hausdorff import similarity
+from shapeAnalysisProject.shape_matching.Hu_moments import similarity
+from shapeAnalysisProject.shape_matching.hausdorff import similarity_hausdorff
```

Comment:
make this as:
`from shapeAnalysisProject.shape_matching.Hu_moments import similarity as similarity_Hu`
`from shapeAnalysisProject.shape_matching.hausdorff import similarity as similarity_hausdorff`